open_humans/templates/member/my-member-settings.html

```diff
@@ -96,7 +96,7 @@ <h5>Allow other members to contact you?</h5>
     <div class="col-md-6">
       <p>{{ user.email }}</p>
 
-      <a href="{% url 'my-member-change-email' %}?next={% url 'my-member-settings' %}"
```

Comment:
I don't think this should be removed, it seems to be breaking this:
https://github.com/OpenHumans/open-humans/blob/master/open_humans/member_views.py#L190